src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/LoginThrottlingFactory.php

```diff
@@ -54,6 +54,7 @@ public function addConfiguration(NodeDefinition $builder)
             ->children()
                 ->scalarNode('limiter')->info(sprintf('A service id implementing "%s".', RequestRateLimiterInterface::class))->end()
                 ->integerNode('max_attempts')->defaultValue(5)->end()
+                ->scalarNode('lock_factory')->info('The service ID of the lock factory used by the login rate limiter (or null to disable locking)')->defaultValue(null)->end()
```

Comment:
Since this is a new option and we add default `null`, it means that we disable locking for all existing users. 

This is a BC break. Was the login throttling released in 5.2?

Reply:
Yes, and as part of the experimental security. So I think this is allowed, but it indeed should be documented as such in the UPGRADE/CHANGELOG

Reply:
Could we find a way to add this without breaking BC?

We should try to avoid BC breaks even in experimental code.

Reply:
... only if it's easy to do so.

Reply:
I think the only way to make this BC is deprecating not setting this option (and setting the default to `lock.factory` again). But I'm -1 on requiring users to do `lock_factory: null`.

Imho, this is exactly why these features are experimental: we include them to get some feedback from real projects and allow ourselves to improve the feature based on this project. After speaking with Jordi, I think it makes much more sense to not lock for login throttling by default.

Reply:
Fair enough. Im happy we discussed this. 

Make sure to update UPGRADE-5.3.md.